xtask/src/examples.rs

```diff
@@ -224,23 +217,41 @@ impl OakExample for FunctionsExample<'_> {
         }
     }
 
-    fn construct_example_server_run_step(&self, run_clients: Step) -> Step {
+    fn construct_server_run_step(&self, run_clients: Step) -> Step {
         let app_variant = self.options.application_variant.as_str();
         match self.applications.get(app_variant) {
             None => run_clients,
-            Some(app) => app.construct_example_server_run_step(self, run_clients),
+            Some(app) => app.construct_server_run_step(self, run_clients),
+        }
+    }
+
+    fn get_server_manifest(server_variant: &ServerVariant) -> &str {
```

Comment:
(nit) the RHS should be `&'static str`, otherwise currently the ref is probably taken to mean to have the safe lifetime as `&ServerVariant` (which works, but it's semantically incorrect).

Reply:
Thank you, done!